### PR TITLE
Release 0.16.0

### PR DESCRIPTION
**Pull request by Kavindu-Dodan**
The PR contains release notes for 0.16.0 and updated the README.md

### CODE REVIEW

File: releasenotes/releasenotes_V0.16.0.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+- No new features
+
+## Fixed Issues
+
+- No fixed issues
+
+## Other Changes
+
+The dynatrace-service has absorbed the functionality of the dynatrace-sli-service.
```
Comment (agrimmer) [inline releasenotes/releasenotes_V0.16.0.md:13]: Ticket number was https://github.com/keptn-contrib/dynatrace-service/issues/363 if you like to add this info.

**Review by agrimmer — DISMISSED** [review]
LGTM

File: releasenotes/releasenotes_V0.16.0.md (hunk 1 above)
Comment (Kavindu-Dodan) [reply releasenotes/releasenotes_V0.16.0.md:13]: Yes, I will add this as it's an important detail.

Comment (Kavindu-Dodan) [reply releasenotes/releasenotes_V0.16.0.md]: Updated with the latest push

**Review by agrimmer — APPROVED** [review]
LGTM

**Pull request merged**